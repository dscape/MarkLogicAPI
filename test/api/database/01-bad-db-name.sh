#!/bin/sh
curl -svX PUT localhost:6984/TestShouldFail -d "" 2>&1 \
  | sed -e 's/^\*.*//' \
  | grep '[<|{].[A-C|E-Z|a-z]'