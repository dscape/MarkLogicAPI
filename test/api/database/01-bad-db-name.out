< HTTP/1.1 400 Bad Request
< Cache-Control: must-revalidate
< Content-type: text/plain; charset=UTF-8
< Server: MarkLogic
< Content-Length: 165
< Connection: close
{"error":"illegal_database_name","reason":"Only lowercase characters (a-z), digits (0-9), and any of the characters _, and - are allowed. Must begin with a letter."}
